cdap-docs/vars

```diff
@@ -27,11 +27,12 @@ BUILD_RST_HASH_LOCATION="cdap-docs/vars"
 BUILD_RST_HASH="039fc5b9a1decc5c67f17c332413f5d9"
 
 # This needs to be explicitly set as Git has trouble identifying it.
-GIT_BRANCH_PARENT="develop"
+GIT_BRANCH_PARENT="release/3.5"
 
 # Used by cdap-apps for Hydrator documentation
-GIT_BRANCH_CASK_HYDRATOR="develop"
+GIT_BRANCH_CASK_HYDRATOR="release/1.4"
+GIT_BRANCH_CASK_TRACKER="release/0.2"
 
 # Used by developers-manual
-CDAP_CLIENTS_RELEASE_VERSION="1.2.0"
+CDAP_CLIENTS_RELEASE_VERSION="1.3.0"
```

Comment:
You're saying that any version other than "1.3" would make it unusable?
How is it working with 1.2 right now then?